document/core/exec/numerics.rst

```diff
@@ -1967,3 +1970,193 @@ Conversions
    \begin{array}{lll@{\qquad}l}
    \narrowu_{M,N}(i) &=& \satu_N(\signed_M(i))
    \end{array}
+
+.. _relaxed-ops:
+
+Relaxed operations
+~~~~~~~~~~~~~~~~~~
+
+The result of *relaxed* operators are *host-dependent*, because the set of
+possible results may depend on properties of the host environment (such as
+hardware). Technically, each such operator produces a fixed-size *list of sets*
+of allowed values.  For each execution of the operator in the same environment,
+only values from the set at the same position in the list are returned, i.e.,
+each environment globally chooses a fixed projection for each operator.
+
+.. note::
+   Each operator can be thought of as a family of operations that is fixed to
+   one particular choice by the execution environment. The fixed operation
+   itself can still be non-deterministic or partial.
+
+
+.. _op-relaxed_madd:
+.. _op-fma:
+
+:math:`\relaxedmadd_N(z_1, z_2, z_3)`
+.....................................
+
+Relaxed multiply-add allows for fused or unfused results. The function :math:`\fma`
+is the same as *fusedMultiplyAdd* defined by |IEEE754|_ (Section 5.4.1).
+
+.. math::
+   \begin{array}{@{}llcll}
+   \EXPROFDET & \relaxedmadd_N(z_1, z_2, z_3) &=& [ \fadd_N(\fmul_N(z_1, z_2), z_3), \fma_N(z_1, z_2, z_3) ] \\
+   & \relaxedmadd_N(z_1, z_2, z_3) &=& \fma_N(z_1, z_2, z_3) \\
+   \\
+   & \fma_N(z_1, z_2, z_3) &=& \ieee_N(z_1 \cdot z_2 + z_3) \\
```

Comment:
Ah, sorry, I was a bit quick to approve. This of course needs to specify all the cases for NaNs and infinities, in the same manner as fadd etc.

(Also, since I didn't point it out explicitly before, please don't forget that all the descriptions here still need a proper prose counterpart.)

Reply:
Yea, I missed it too, ptal #135.